HelloTill/loanpro-data-ingestion
# RL-14: Integrate LoanPro API

Adds a basic connection to LoanPro API and pulls loans.

To test:
1. yarn install
1. ask me for the env variables you need
1. yarn start
1. POST localhost:3000/v1/functions/loanpro-data-ingestion-dev-loanpro-data-ingestion/invocations with any JSON body
1. Verify that the response is a list of loans from loan pro
